### ISSUE TITLE
q-input type:password Extra icons

### ISSUE THREAD

**Issue by BeginsDuang** [quasarframework/quasar#7213]
```html
<q-input v-model="password" filled :type="isPwd ? 'password' : 'text'" hint="Password with toggle">
        <template v-slot:append>
          <q-icon
            :name="isPwd ? 'visibility_off' : 'visibility'"
            class="cursor-pointer"
            @click="isPwd = !isPwd"
          />
        </template>
</q-input>
```
![1](https://user-images.githubusercontent.com/25838823/84565667-262a5280-ad9d-11ea-8bdb-7fc6aba08f49.gif)
[doc example](https://quasar.dev/vue-components/input)

**Comment by pdanpdan** [comment]
Please fill the informations required in the issue template. I would guess what your browser is, but it would only lose time.

**Comment by BeginsDuang** [comment]
@pdanpdan I am sorry，brower：Microsoft Edge（83.0.478.45）
Google Chrome(83.0.4103.97) It's working

**Comment by pdanpdan** [comment]
:) in it's short history Edge managed to add something:
https://stackoverflow.com/questions/61449079/how-to-hide-the-eye-from-a-password-input-in-ms-edge-and-ie

**Comment by BeginsDuang** [comment]
thanks it's working
```css
input::-ms-reveal, input::-ms-clear {
    display: none;
}
```

**Comment by pdanpdan** [comment]
I'll leave this open because it looks like the new edge also implements this.

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.12.5.
